tests/mentorship_relation/test_api_cancel_relation.py

```diff
@@ -61,7 +77,61 @@ def test__mentee_cancel_mentorship_relation(self):
             self.assertEqual(MentorshipRelationState.CANCELLED, self.mentorship_relation.state)
             self.assertDictEqual(messages.MENTORSHIP_RELATION_WAS_CANCELLED_SUCCESSFULLY,
                              json.loads(response.data))
-
+    
+    #missing authorisation token
+    
+    def test__mentor_cancel_token_missing(self):
+        self.assertEqual(MentorshipRelationState.ACCEPTED, self.mentorship_relation.state)
+        with self.client:
+            response = self.client.put('/mentorship_relation/%s/cancel' % self.mentorship_relation.id) 
+            self.assertEqual(401, response.status_code)
+            self.assertEqual(MentorshipRelationState.ACCEPTED, self.mentorship_relation.state)
+            self.assertDictEqual(messages.AUTHORISATION_TOKEN_IS_MISSING, 
+                                 json.loads(response.data))
+            
+    def test__mentee_cancel_token_missing(self):
```

Comment:
This test and the test above are exactly equal, so this one can be deleted.
Can you please delete this?